pkgs/development/tools/analysis/svlint/default.nix

```diff
@@ -5,16 +5,18 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "svlint";
-  version = "0.5.1";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "dalance";
     repo = "svlint";
     rev = "v${version}";
-    sha256 = "sha256-BgkzbKRcZkot3qkwPqSE9QkH3A3HNDuLjpFzKsU+Wb0=";
+    sha256 = "sha256-dtfOSj0WnNyQLimXkSK+L8pWL/oc0nIugDyUmGaBP3w=";
   };
 
-  cargoSha256 = "sha256-HeFh8H7IN3m4HiEH1QbCBROslzVCzYxGIaeyM4K7gcs=";
+  cargoPatches = [ ./add-cargo-lock.patch ];
```

Comment:
Please commit the file plain without the patch on top. I think we can also use that direct.

Reply:
Good idea.